NFT-Sample/index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="main.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Outfit:wght@300;400;600&display=swap" rel="stylesheet"> 
+    <title>Document</title>
+</head>
+<body>
+    
+    <div class="card-container">
+    <div class="main-card">
+        <div class="image1"></div>
```

Comment:
it's better to set the image here in HTML rather than in CSS. to further improve, you can use an HTML semantic instead. you can try to search for `<article>` and `<figure>`